frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateConfigureStep.tsx

```diff
@@ -122,15 +124,21 @@ export const SiteChooser = (): JSX.Element => {
                     <h2>Select where you want to track events from.</h2>
                     {hasSnippetEventsLoading ? (
                         <Spinner />
-                    ) : snippetHosts.length > 0 ? (
+                    ) : combinedSnippetAndLiveEventsHosts.length > 0 ? (
                         <>
                             <p>
-                                Not seeing the site you want?{' '}
-                                <Link onClick={() => setStepKey(OnboardingStepKey.INSTALL)}>Install posthog-js</Link> or
+                                Not seeing the site you want? Try clikcing around on your site to trigger a few events.
+                                If you haven't yet,{' '}
+                                <Link onClick={() => setStepKey(OnboardingStepKey.INSTALL)}>install posthog-js</Link> or
                                 the HTML snippet wherever you want to track events, then come back here.
                             </p>
+                            {isCloud && (
+                                <p className="text-muted italic">
+                                    Note: Sites must be served over HTTPS to be selected.
+                                </p>
+                            )}
                             <div className="space-y-2">
-                                {snippetHosts.map((host) => (
+                                {combinedSnippetAndLiveEventsHosts.concat('https://posthog.com').map((host) => (
```

Comment:
I'm not sure I understand why you added this.

Reply:
Ha oops that wasn't supposed to be there